src/browser-pool/webdriver-pool.ts

```diff
@@ -17,13 +17,13 @@ export class WebdriverPool {
     }
 
     async getWebdriver(
-        browserName: SupportedBrowser,
-        browserVersion: string,
+        browserName?: string,
+        browserVersion?: string,
         { debug = false } = {},
     ): ReturnType<typeof this.createWebdriverProcess> {
-        const driverName = getDriverNameForBrowserName(browserName);
+        const browserNameNormalized = getNormalizedBrowserName(browserName);
 
-        if (!driverName) {
+        if (!browserNameNormalized) {
             throw new Error(
```

Comment:
As we now dont want to map "browserName" to "driverName", we still need to check if "browserName" is valid (because this method is called inside NewBrowser with its config's browserName, which is user provided value, which can be invalid)